atecc608a_utils.h

```diff
@@ -61,4 +61,7 @@ psa_status_t atecc608a_get_serial_number(uint8_t *buffer, size_t buffer_size,
 
 psa_status_t atecc608a_check_config_locked();
 
+/** Generate a 32 byte random number from the CryptoAuth device. */
+psa_status_t atecc608a_random(uint8_t *rand_out);
```

Comment:
A buffer without a length is pretty fragile. Call the function `atecc608a_random_32_bytes`?

Reply:
Done.